dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -140,65 +140,80 @@ export const Columns: React.FC<{
 		role="menubar"
 		data-cy="nav-menu-columns"
 	>
-		{nav.pillars.map((column, i) => (
-			<Column
-				column={column}
-				key={column.title.toLowerCase()}
-				index={i}
-				isLastColumn={i !== nav.pillars.length - 1}
-			/>
-		))}
-
-		<li>
-			<ThemeProvider theme={{ ...buttonThemeBrand }}>
-				<div css={searchBar}>
-					<LinkButton
-						href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
-						tabIndex={-1}
-						className="selectableMenuItem"
-						priority="secondary"
-						icon={
-							<SvgMagnifyingGlass
-								isAnnouncedByScreenReader={true}
-								size="medium"
-							/>
-						}
-						aria-label="Search with google"
-						data-link-name="nav2 : search : submit"
-						type="submit"
-					>
-						Search
-					</LinkButton>
-				</div>
-			</ThemeProvider>
+		<>
```

Comment:
Is this fragment required? I was under the impression you only needed to use `<>` if you needed to return multiple JSX nodes from a component without wrapping them in something else.